client/app/bundles/shared/components/Footer/Footer.jsx

```diff
@@ -1,43 +1,86 @@
+// @flow
 import React from 'react';
+import { IntlProvider, injectIntl } from 'react-intl';
+import { defaultMessages, defaultLocale } from 'libs/i18n/default';
+import { getMessages } from 'libs/i18n/I18nUtils';
 import css from './Footer.scss';
 import Resources from './Resources';
 import Connect from './Connect';
 import Ifme from './Ifme';
 import DropdownGhostSmall from '../Dropdown/DropdownGhostSmall';
 
-import enYML from '../../../../../../config/locales/en.yml';
+const we = defaultMessages.sharedFooterLicenseWe;
+const foss = defaultMessages.sharedFooterLicenseFoss;
+const licenseSubtitle = defaultMessages.sharedFooterLicenseSubtitle;
+const licenseName = defaultMessages.sharedFooterLicenceName;
 
-const navProp = enYML.en.navigation;
-const we = enYML.en.shared.footer.license_we;
-const foss = enYML.en.shared.footer.license_foss;
+type FooterProps = {
+  intl: Object,
+  onChange: (locale: string) => void,
+}
 
-const Footer = () => (
-  <div className={css.footer}>
-    <div id={css.footer_contest}>
-      <div className={css.table}>
-        <div className={css.row}>
-          <div className={`${css.table_cell} ${css.if_me}`}>
-            <Ifme app_name={enYML.en.app_name} navigation={navProp} />
-          </div>
-          <div className={`${css.table_cell} ${css.connect}`}>
-            <Connect navigation={navProp} common={enYML.en.common} />
-          </div>
-          <div className={`${css.table_cell} ${css.resources}`}>
-            <Resources pages={enYML.en.pages.resources} navigation={navProp} />
-          </div>
-          <div className={`${css.table_cell} ${css.dropdown}`}>
-            <DropdownGhostSmall />
-          </div>
-          <div className={`${css.table_cell} ${css.love_foss}`}>
-            <h4>{we} &hearts; {foss}</h4>
-            <a className={css.license} href="https://github.com/ifmeorg/ifme/blob/master/LICENSE.txt" target="blank" >
-              {enYML.en.shared.footer.license_subtitle} {enYML.en.shared.footer.licence_name}
-            </a>
+const InjectedFooter = injectIntl(({ intl, onChange }: FooterProps) => {
+  const { formatMessage } = intl;
+  return (
+    <div className={css.footer}>
+      <div id={css.footer_contest}>
```

Comment:
Can combine `id` attribute into the immediate parent div. In the corresponding `Footer.scss`, the `position: relative` can be part of the `.footer` class.